cdap-app-templates/cdap-etl/cdap-etl-batch-template/pom.xml

```diff
@@ -26,12 +26,12 @@
     <version>3.2.0-SNAPSHOT</version>
   </parent>
 
-  <artifactId>cdap-etl-batch-app</artifactId>
-  <name>CDAP ETL Batch Application</name>
+  <artifactId>cdap-etl-batch-template</artifactId>
```

Comment:
I thought we are going to rename cdap-etl-batch-app to cdap-etl-batch? And not cdap-etl-batch-template?